serializer.rb

```diff
@@ -0,0 +1,60 @@
+require 'json'
+
+class Serializer
+  def initialize(path = './data')
+    @path = path
+  end
+
+  def load(file_name)
+    data = []
+    begin
+      deserialized_data = JSON.load_file("#{@path}/#{file_name}.json")
+      deserialized_data.each do |object|
+        case file_name
+        when 'books' then data << Book.new(object['title'], object['author'])
+        when 'students'
+          data << Student.new(object['age'], object['id'], object['name'],
+                              parent_permission: object['parent_permission'])
```

Comment:
- [ ] Same error is raised when I saved student data and try to re-run the App, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/217555948-cb9c68b5-5384-4c3e-9400-df56794f79f0.png)

[Hint] The problem is fixed by rearranging the student arguments and switching places between id and name.